mautrix_instagram/web/provisioning_api.py

```diff
@@ -136,7 +136,16 @@ async def login(self, request: web.Request) -> web.Response:
                 "response": e.body.serialize(),
             }, status=202, headers=self._acao_headers)
         except IGCheckpointError as e:
-            await api.challenge_auto(reset=True)
+            try:
+                await api.challenge_auto(reset=True)
+            except Exception as e:
```

Comment:
A more specific exception would be nicer, but I guess this works too

Reply:
I think it's like a JSON decode error, but I'm not really sure. Maybe I'll add some logging here so we can investigate further and narrow down whenever we see another instance.